align_utils.py

```diff
@@ -0,0 +1,63 @@
+import itertools
```

Comment:
I'm not sure what to do with this, but maybe it can be deleted if it's not used anywhere?